app/assets/stylesheets/application.css

```diff
@@ -68,3 +105,19 @@ p {
 h1 {
   color: white
 }
+.reviewForm{
+  margin-top: 20rem;
+  font-style: italic;
+  color: red;
+  font-size: 1rem;
+}
+.submitButton{
+  color: black;
+}
+.forgotPassForm{
+  margin-top: 3rem;
+  margin-left: 2rem;
+  font-style: italic;
+  color: #ffffff;
+  font-size: 4rem;
+}
```

Comment:
Your main styling shouldn't be in the `application.css` file: it should be separated out into its own file that will automatically be inherited by the asset pipeline (i.e., have a file for `review_form.css`, etc)